### PR TITLE
feat: implement transfer box and no gas error message

### PR DESCRIPTION
**Pull request by rodrigopavezi**

### CODE REVIEW

File: src/transaction/transaction-box.test.tsx [hunk 1]
```diff
@@ -0,0 +1,28 @@
+import * as React from 'react';
+import { create } from 'react-test-renderer';
+import { TransactionBox } from './transaction-box';
+
+test('should show Transaction Box with ETH Icon', () => {
```
Comment (sk91) [inline src/transaction/transaction-box.test.tsx]: I've been using the most commonly accepted syntax `describe() -> it()` let's stick to it

Comment (sk91) [reply src/transaction/transaction-box.test.tsx]: also, test file name should be '.spec.tsx'

Comment (rodrigopavezi) [reply src/transaction/transaction-box.test.tsx]: @sk91 yeap, I was wondering about that as well but when I went to the jest documentation they stop using `describe() -> it()` so I think for good reference for devs I think we should align our tests with the documentation. Thoughts?

**Comment by sk91** [comment]
Thous are aliases, that allow the same, and it is more universally accepted.

Plus if we use different e2e framework (which we do now) it will be good
for the api usage to be aligned

**Pull request merged**